etc/sample_data/eventlog.xml

```diff
@@ -392,7 +392,8 @@
+<!-- below was manually edited: 4164 is a PID collision with the dropper. changed to 14164 -->
+<Event xmlns='http://schemas.microsoft.com/win/2004/08/events/event'><System><Provider Name='Microsoft-Windows-Sysmon' Guid='{5770385F-C22A-43E0-BF4C-06F5698FFBD9}'/><EventID>1</EventID><Version>5</Version><Level>4</Level><Task>1</Task><Opcode>0</Opcode><Keywords>0x8000000000000000</Keywords><TimeCreated SystemTime='2019-07-24T18:02:39.408631400Z'/><EventRecordID>394</EventRecordID><Correlation/><Execution ProcessID='3324' ThreadID='3220'/><Channel>Microsoft-Windows-Sysmon/Operational</Channel><Computer>DESKTOP-FVSHABR</Computer><Security UserID='S-1-5-18'/></System><EventData><Data Name='RuleName'></Data><Data Name='UtcTime'>2019-07-24 18:02:39.406</Data><Data Name='ProcessGuid'>{87E8D3BD-9D3F-5D38-0000-00109A641700}</Data><Data Name='ProcessId'>14164</Data><Data Name='Image'>C:\Program Files (x86)\Google\Update\GoogleUpdate.exe</Data><Data Name='FileVersion'>1.3.34.11</Data><Data Name='Description'>Google Installer</Data><Data Name='Product'>Google Update</Data><Data Name='Company'>Google LLC</Data><Data Name='OriginalFileName'>GoogleUpdate.exe</Data><Data Name='CommandLine'>"C:\Program Files (x86)\Google\Update\GoogleUpdate.exe" /registermsihelper</Data><Data Name='CurrentDirectory'>C:\Program Files (x86)\Google\Update\1.3.34.11\</Data><Data Name='User'>NT AUTHORITY\SYSTEM</Data><Data Name='LogonGuid'>{87E8D3BD-998D-5D38-0000-0020E7030000}</Data><Data Name='LogonId'>0x3e7</Data><Data Name='TerminalSessionId'>0</Data><Data Name='IntegrityLevel'>System</Data><Data Name='Hashes'>MD5=82F657B0AEE67A6A560321CF0927F9F7,SHA256=794CF7644115198DB451431BCA7C89FF9A97550482B1E3F7F13EB7ACA6120A11</Data><Data Name='ParentProcessGuid'>{87E8D3BD-9D3F-5D38-0000-00103C561700}</Data><Data Name='ParentProcessId'>3328</Data><Data Name='ParentImage'>C:\Program Files (x86)\Google\Update\GoogleUpdate.exe</Data><Data Name='ParentCommandLine'>"C:\Program Files (x86)\Google\Update\GoogleUpdate.exe" /ua /installsource scheduler</Data></EventData></Event>

```

Comment:
The collision is not within this file?

Reply:
![image](https://user-images.githubusercontent.com/69007229/97932381-b2c1b700-1d24-11eb-9ad8-2dc23bb0cfad.png)
it is, just lower than the fiels-changed view shows